### PR TITLE
Fixes some styles and variables

### PR DESCRIPTION
**Pull request by yanas**

### CODE REVIEW

File: css/_toolbars.scss [hunk 1]
```diff
@@ -19,74 +19,6 @@
...
-    line-height: 50px !important;
-    position: relative;
-    text-align: center;
-    top:0px;
-    vertical-align: middle;
-    width: 50px;
-
-    &_hangup {
-        color: $hangupColor;
-        font-size: $hangupFontSize !important;
```
Comment (virtuacoplenny) [inline css/_toolbars.scss]: 👍 Getting rid of !important is such a big win.

File: css/themes/_light.scss [hunk 2]
```diff
@@ -91,7 +91,17 @@ $popoverFontColor: #ffffff;
 $popupMenuSelectedItemBackground: rgba(256, 256, 256, .2);
 
 // Toolbar
+$secToolbarBackground: rgba(0, 0, 0, 0.5);
```
Comment (virtuacoplenny) [inline css/themes/_light.scss]: Is secToolbar a common abbreviation in jitsi-meet?

File: css/themes/_light.scss [hunk 3]
```diff
@@ -91,7 +91,17 @@ $popoverFontColor: #ffffff;
...
+$sideBarBackground: rgba(0, 0, 0, 0.75);
 $splitterColor: #ccc;
+$toolbarBackground: rgba(0, 0, 0, 0.5);
+$toolbarBadgeBackground: #165ECC;
+$toolbarBadgeColor: #FFFFFF;
+$toolbarButtonColor: #FFFFFF;
+$toolbarSelectBackground: rgba(0, 0, 0, .6);
+$toolbarTitleColor: #FFFFFF;
+$toolbarToggleBackground: #12499C;
```
Comment (virtuacoplenny) [inline css/themes/_light.scss]: I don't fully understand the distinction between variables.scss and a themes scss file. It seems like both have variables and both are necessary for scss to compile. What decides when a variables goes into one or the other?

File: css/_side_toolbar_container.scss [hunk 4]
```diff
@@ -2,7 +2,7 @@
  * Toolbar side panel main container element.
  */
 #sideToolbarContainer {
-    background-color: rgba(0,0,0,0.8);
+    background-color: $sideBarBackground;
```
Comment (virtuacoplenny) [inline css/_side_toolbar_container.scss]: The id says sideToolbar but the variable is sideBar. Should the naming be consistent? What is a side toolbar?

File: css/themes/_light.scss (hunk 2 above)
Comment (yanas) [inline css/themes/_light.scss]: SecondaryToolbar is the class

File: css/themes/_light.scss (hunk 3 above)
Comment (yanas) [inline css/themes/_light.scss]: One was created only to change the color scheme, but then I see that we have used the variables file to add colours as well, so currently it's a mess.

File: css/_side_toolbar_container.scss (hunk 4 above)
Comment (yanas) [inline css/_side_toolbar_container.scss]: When moving the side toolbar to react it was renamed to SecondaryToolbar, but the styles weren't changed, so now there's inconsistency in the names. I can change my var to secToolbarContainerBackground, but I'm not sure if this is an improvement. I don't want to fix styling names of existing code right now.

**Pull request merged**